apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.Tests/RetryingMutateRowsTest.cs

```diff
@@ -183,6 +186,88 @@ public class RetryingMutateRowsTest
             Assert.Equal((int)Code.DeadlineExceeded, entry.Status.Code);
         }
 
-        // TODO: Test with a mock IClock to make sure we don't go over the total timeout when retrying
+        [Fact]
+        public async Task RetryAfterTotalExpiration()
+        {
+            var settings = new BigtableServiceApiSettings();
+            // Don't allow for any time to retry.
+            settings.MutateRowsRetrySettings =
+                settings.MutateRowsRetrySettings.WithTotalExpiration(
```

Comment:
With a total expiration of 0, I'm not sure I'd expect it to make any requests at all. Would it be better to have a total expiration that is "over" by the time the first response comes back?